backend/internal/database/database.go

```diff
@@ -0,0 +1,57 @@
+package database
+
+import (
+	"github.com/ibeckermayer/teleport-interview/backend/internal/model"
+	"github.com/jmoiron/sqlx"
+	"github.com/pborman/uuid"
+)
+
+// Config is a database config object
+type Config struct {
+	Env string // Determines whether the production or development database is created/used; if \"dev\", the app will seed the database with sample data for manual testing.
```

Comment:
nit: put the comment above the field, not on the same line
also, wrap the long comment over multiple lines